### ISSUE TITLE
Not moving any fort only show [MainThread] [ShowBestPokemon] [INFO] *Best Pokemons*

### ISSUE THREAD

**Issue by malaypatel11** [PokemonGoF/PokemonGo-Bot#5776]
### Expected Behavior
Not move any fort 

### Actual Behavior
[2016-10-09 14:11:59] [MainThread] [ShowBestPokemon] [INFO] *Best Pokemons* (Charmander, CP 12, IVCP 0.66, DPS 15.12, HP 10/10)
[2016-10-09 14:12:09] [MainThread] [RecycleItems] [INFO] Forcing item recycle based on schedule
[2016-10-09 14:12:09] [MainThread] [RecycleItems] [INFO] Next forced item recycle at 14:14:47

### Your FULL config.json (remove your username, password, gmapkey and any other private info)

http://pastebin.com/L6L29mTj

### Other Information
OS: 
mac
Branch: 
dev

Git Commit: 
5ae0afc9e770cc3b6d6bdc31170fa080ed4a259b

Python Version: 
Python 2.7.10

**Comment by mmixx** [comment]
Bot is currently not functioning because of the 0.39 update... I myself, also waits for the pogo api devs to update their API from 0.35 to 0.39... Also, see the other threads for more info... #5773 #5768

**Comment by JWNS** [comment]
Still not moving for me.
The API has already been cracked down no ?
What did i do wrong ? =/

**Comment by MZorzy** [comment]
@jwns have u git pull to last master/dev ?

**Comment by JWNS** [comment]
@MZorzy Yup, I recloned the project and did the whole installation process from the begining a few hours ago =(
